### ISSUE TITLE
Unable to switch video from Youtube to Youtube

### ISSUE THREAD

**Issue by MikeNized** [calzoneman/sync#283]
Location : http://synchtube.6irc.net/r/SomethingTestPonMarkII
Firefox 24.0

I need to describe what I tested on it...
1 - Two videos are in the Playlist, the two are hosted on Youtube.
2 - Watching the first video works perfectly fine...
3 - The problem is that when the video reachs the end, the playlist go to the next (also hosted on Youtube) but doesn't update the leader or the watchers and loop back on the previous video.
4 - It goes back to normal once you hit "Refresh media".

Which means each time a youtube video ends, the host must press "Refresh Media" to get the proper video on his side and allowing the rest of them to watch it.

What is strange is the time is still correctly synched : 
- It is supposed to stream Video B but the leader is watching Video A at a time of 1:23
- If the watchers (not the leader) hit "Refresh Media", the video B (supposed to be the current video in the playlist) is correctly time-synched with the video that the leader is watching (video A).

In other words, even if we press "Play" to a next video, if it is still Youtube, the video doesn't switch but the time is synched to the start.

The problem doesn't appear when we switch to another source (youtube->dailymotion, for example).
Switching browsers didn't solved the problem, disable the extensions either.

Please contact me for more informations.

**Comment by nuclearace** [comment]
This was/is a problem with youtube. They should be fixing it now.

**Comment by nuclearace** [comment]
https://code.google.com/p/gdata-issues/issues/detail?id=5273

**Comment by MikeNized** [comment]
Alright, thanks for the tip. Waiting for it, then.

**Comment by calzoneman** [comment]
Ah yes, I forgot to announce this issue on 6irc.  YouTube has informed me that they're rolling out the patch, so you can try clearing your cache and refreshing.
